addons/l10n_lv/i18n/l10n_lv.pot

```diff
@@ -0,0 +1,243 @@
+# Translation of Odoo Server.
```

Comment:
For localisation especially, a pot without it's po file is useless. So you will need to do the po in latvia :smile: To do this you can do it from the interface by installing the lang you want to export then going in the settings -> translation -> export translation -> select the language and the app you want to export and hop you can download the po file